clifford/_multivector.py

```diff
@@ -745,6 +749,52 @@ def normalInv(self, check=True) -> 'MultiVector':
         """
         return self._pick_inv(fallback=False if check else None)
 
+    def hitzer_inverse(self):
+        """
+        Performs the inversion operation as described in the paper:
+        Multivector and multivector matrix inverses in real Clifford algebras
+        Eckhard Hitzer, Stephen Sangwine (2017)
+        """
+        tot = sum(self.layout.sig != 0)
+        if tot <= 5:
+            if tot == 1:
+                # Equation 4.3
+                mv_invol = self.gradeInvol()
+                numerator = mv_invol
+                denominator = (self * mv_invol)[()]
+            elif tot == 2:
+                # Equation 5.5
+                mv_conj = self.conjugate()
+                numerator = mv_conj
```

Comment:
This case and the one above is just normalInv, isn't it?

Reply:
yeah good point, I think they are the same